### PR TITLE
#276: fix duplicated PullRequestReviewEvent events

### PR DESCRIPTION
**Pull request by Yegorov**
@yegor256 check, please

### CODE REVIEW

File: judges/github-events/github-events.rb [hunk 1]
```diff
@@ -133,6 +133,14 @@ def self.fill_up_event(fact, json)
       case json[:payload][:action]
       when 'created'
         skip_event(json) if json[:payload][:pull_request][:user][:id].to_i == fact.who
+        if Fbe.fb.query(
+          "(and (eq repository #{fact.repository}) " \
+          '(eq what "pull-was-reviewed") ' \
+          "(eq who #{fact.who}) " \
+          "(eq issue #{json[:payload][:pull_request][:number]}))"
+        ).each.last
```
Comment (yegor256) [inline judges/github-events/github-events.rb:141]: @Yegorov this looks like a pretty dirty hack :) Maybe we can better analyze the incoming JSON (from GitHub) and understand the difference between two events? They send us them both, for some reason. We just need to understand the reason.

**Review by yegor256 — CHANGES_REQUESTED** [review]
@Yegorov I suggest we try a better solution, more aligned with the logic of GitHub API

File: judges/github-events/github-events.rb (hunk 1 above)
Comment (Yegorov) [reply judges/github-events/github-events.rb:141]: @yegor256 not sure, differences only in `id`. Look [this diff](https://www.diffchecker.com/1DcBl2f5/)

**Comment by yegor256** [comment]
@Yegorov this is weird, but we can't do anything about it. Thanks!

**Comment by 0crat** [comment]
@yegor256 Thanks for the review! You've earned +5 points for this: +15 as a basis; -10 for very few (1) comments.  Your running balance is [-546](https://zerocracy.github.io/judges-action/zerocracy-vitals.html).

**Comment by 0crat** [comment]
@Yegorov Thanks for your contribution! You've earned **15 points**: +20 base, +20 for reviews, +10 for 200 hits-of-code (capped at 30), -15 for no code review, -5 for few comments, -15 for 800+ hits-of-code. Remember, more [code reviews](https://www.yegor256.com/2014/11/14/hits-of-code.html) and comments can boost your score. [Keep](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) the contributions coming! Your balance is now [+314](https://zerocracy.github.io/judges-action/zerocracy-vitals.html).

**Pull request merged**